apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/name-step/index.tsx

```diff
@@ -48,13 +52,19 @@ const NameStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } )
 						autoCorrect="off"
 						data-hj-whitelist
 					/>
-					<p className="nux-launch-step__input-hint">
+					<div className="nux-launch-step__input-hint">
 						<Tip size={ 18 } />
 						{ /* translators: The "it" here refers to the site title. */ }
 						<span>{ __( "Don't worry, you can change it later.", 'full-site-editing' ) }</span>
-					</p>
+					</div>
```

Comment:
This fixes the nested `<p><p></p></p>` warning as the `<Tip>` component uses `<p>` as well.